zephyr/wrapper.c

```diff
@@ -600,6 +601,10 @@ int task_main_start(struct sof *sof)
 		sys_comp_crossover_init();
 	}
 
+	if (IS_ENABLED(CONFIG_COMP_DRC)) {
+		sys_comp_drc_init();
+	}
+
```

Comment:
I don't understand this change. Do you enable DRC for Zephyr? Then it should be a separate commit with proper explanation.